code/game/objects/items/wiki_manuals.dm

```diff
@@ -37,9 +11,11 @@
 	if(!wiki_url)
 		user.balloon_alert(user, "this book is empty!")
 		return
-
-	credit_book_to_reader(user)
-	DIRECT_OUTPUT(user, browse(WIKI_PAGE_IFRAME(wiki_url, page_link), "window=manual;size=[BOOK_WINDOW_BROWSE_SIZE]")) // if you change this GUARANTEE that it works.
+	else
+		credit_book_to_reader(user)
+		if(tgui_alert(user, "This will open the wiki page of the book you are reading in your browser. Are you sure?",, list("Yes","No"))!="Yes")
+			return
+		usr << link("[wiki_url]/[page_link]")
```

Comment:
```suggestion
	credit_book_to_reader(user)
	if(tgui_alert(user, "This will open the wiki page of the book you are reading in your browser. Are you sure?",, list("Yes","No"))!="Yes")
		return
	usr << link("[wiki_url]/[page_link]")
```

indentation is not useful here since we already early return on line 13